### PR TITLE
bugs fixex and ui changes

### PR DESCRIPTION
**Pull request by Stepanwdr**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
 </head>
 
 <body>
-	<noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (armanist) [inline public/index.html:21]: What is the change here?

File: src/assets/locales/am/common.json [hunk 2]
```diff
@@ -1,56 +1,56 @@
-{
```
Comment (armanist) [inline src/assets/locales/am/common.json:1]: can not see any difference here

File: src/assets/locales/am/validation.json [hunk 3]
```diff
@@ -1,44 +1,44 @@
-{
```
Comment (armanist) [inline src/assets/locales/am/validation.json:1]: can not see any difference here

File: src/assets/locales/en/common.json [hunk 4]
```diff
@@ -1,58 +1,58 @@
-{
```
Comment (armanist) [inline src/assets/locales/en/common.json:1]: can not see any difference here

File: src/components/Auth/ActivatePage.jsx [hunk 5]
```diff
@@ -1,24 +1,24 @@
-import { useEffect } from 'react'
```
Comment (armanist) [inline src/components/Auth/ActivatePage.jsx:1]: can not see any difference here

File: src/components/Auth/ActivatePage.jsx [hunk 6]
```diff
@@ -1,24 +1,24 @@
-import { useEffect } from 'react'
-import { authApi } from './../../api/authApi';
```
Comment (armanist) [inline src/components/Auth/ActivatePage.jsx:2]: can not see any difference here

**Review by armanist — CHANGES_REQUESTED** [review]
There are lots of files that are not getting actually changed but appeared in git all those files need to be reverted and only the actual changes should be kept.

Also remove `.idea` related files

**Review by hatik92 — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (Stepanwdr) [reply public/index.html:21]: its will not be in the next commit

**Review by armanist — APPROVED** [review]

**Pull request merged**